### PR TITLE
Upgraded RavenDB.Client library to 6.0.104

### PR DESCRIPTION
**Pull request by ebrob**
- Updated RavenDB.Client package to 6.0.14
- Made a few code updates because of constructor changes
- Updated various test packages
- Updated project version
- Confirmed that unit tests pass

### CODE REVIEW

**Comment by ebrob** [comment]
Very strange - there are still a few lines that GitHub shows changed, but I didn't touch at all.

File: src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj
Comment (ryanheath) [inline src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj:1]: Bump to <Version>6.0.104</Version>

File: src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj
Comment (ryanheath) [inline src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj:1]: Bump to <Version>6.0.104</Version>

File: test/Raven.Client.NodaTime.Tests/Raven.Client.NodaTime.Tests.csproj
Comment (ryanheath) [inline test/Raven.Client.NodaTime.Tests/Raven.Client.NodaTime.Tests.csproj:1]: can you upgrade the test assembly references too, like in #32 ?

Comment (ebrob) [reply test/Raven.Client.NodaTime.Tests/Raven.Client.NodaTime.Tests.csproj:1]: Done!

Comment (ebrob) [reply test/Raven.Client.NodaTime.Tests/Raven.Client.NodaTime.Tests.csproj:1]: I re-ran unit tests, they all passed.

File: src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj
Comment (ryanheath) [inline src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj:1]: ```<Version>5.4.1</Version>``` to ```<Version>6.0.104</Version>```

File: src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj
Comment (ryanheath) [reply src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj:1]: ```<Version>5.4.1</Version>``` to ```<Version>6.0.104</Version>```

File: src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj
Comment (ebrob) [reply src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj:1]: Done!

File: src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj
Comment (ebrob) [reply src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj:1]: Done!

**Comment by ryanheath** [comment]
Thanks!

**Pull request merged**